fakturogen-comarch-connector/src/test/java/pl/fakturogen/comarchconnector/converter/InvoiceResponseConverterTest.java

```diff
@@ -0,0 +1,93 @@
+package pl.fakturogen.comarchconnector.converter;
```

Comment:
class looks fine, however I think that it should be moved to /fakturogen/comarch/connector/converter package from current /fakturogen/comarchconnector/converter